test/unit/utils/telemetry/collector_test.rb

```diff
@@ -40,4 +40,14 @@ def test_reset_singleton
     @collector.reset
     assert_equal 0, @collector.list_data_series.count
   end
+
+  def test_telemetry_disabled
+    @collector.load_config(Inspec::Config.mock(telemetry: false))
+    refute @collector.telemetry_enabled?
+  end
+
+  def test_telemetry_enabled
+    @collector.load_config(Inspec::Config.mock(telemetry: true))
+    assert @collector.telemetry_enabled?
```

Comment:
I feel like these tests are pretty silly since I'm just testing that the configuration got loaded and a True/False was set, is there a better way to do this?

Reply:
I would `assert`/`refute` before the `load_config` to show state change. Otherwise, I think these tests are fine.